database/tables/pt_location_networks.pg

```diff
@@ -0,0 +1,10 @@
+columns
```

Comment:
Can you update the `fix-pl-db.sh` in the PrairieTest repo to account for these new tables?

Reply:
Good point. Done in https://github.com/PrairieLearnInc/PrairieTest/pull/776